uk_bin_collection/uk_bin_collection/councils/HuntingdonDistrictCouncil.py

```
@@ -10,49 +14,86 @@
+            page: Unused (maintained for interface compatibility).
+            **kwargs: Must contain 'uprn' (Unique Property Reference Number).
 
+        Returns:
+            dict: Dictionary with 'bins' key containing list of bin types and dates.
+
+        Raises:
+            ValueError: If UPRN is invalid or page cannot be fetched/parsed.
+        """
         try:
             user_uprn = kwargs.get("uprn")
             check_uprn(user_uprn)
```

Ruff (0.14.14)

[error] `check_uprn` may be undefined, or defined from star imports

(F405)